### ISSUE TITLE
EXEC CICS GET CHANNEL marked as error, "RESP expected instead of this input"

### ISSUE THREAD

**Issue by ssamayoa** [IBM/zopeneditor-about#361]
## Development environment used

- Z Open Editor version: 
- Editor Platform
  - [x] Visual Studio Code
  - [ ] Red Hat CodeReady Workspaces
  - [ ] Eclipse Che
  - [ ] Standalone Theia
- Editor Platform Version: 1.83.1
- Operating System : Windows 10 22H2
- Java Version: AdoptOpenJDK 11.0.9.101-hotspot
- [ ] Related to RSE API?
  - RSE API Plugin version:
  - Zowe CLI version:
  - Node.js version:
- Logs attached: no

## Problem Description
EXEC CICS GET CHANNEL marked as error over "CHANNEL", message is "RESP expected instead of this input"

Actual statement on working program:

```
           EXEC CICS GET                                                        
                CHANNEL(HDR-CHANNEL)                                            
                CONTAINER(HDR-CONT-RESP)                                        
                FLENGTH(CONTAINER-LENGTH)                                       
                NODATA                                                          
                RESP(RESP)                                                      
                RESP2(RESP2)                                                    
           END-EXEC.                                                            
```
### Expected behavior
Accept the syntax.

**Comment by phaumer** [comment]
@ssamayoa thanks. The docs tell us to write CONTAINER before CHANNEL: <https://www.ibm.com/docs/en/cics-ts/6.1?topic=summary-get-container-channel> Would that work for you?

**Comment by ssamayoa** [comment]
Thanks for pointing the doc link.

Is not program of mine, is at least a decade old. I did small changes recently and compiler does not complains but plug-in does so I assumed a plug-in fault. Original programmer wrote that way not only on this but in other programs so I can live with the plug-in complain even if compiler does not bothers.

**Comment by phaumer** [comment]
Thanks. We want to fix our parser.

**Comment by phaumer** [comment]
Fixed in v3.4.0.
